src/controller/CHIPDevice.cpp

```diff
@@ -372,9 +374,9 @@ CHIP_ERROR Device::OpenPairingWindow(uint16_t timeout, PairingWindowOption optio
         ReturnErrorOnFailure(
             PASESession::GeneratePASEVerifier(verifier, kPBKDFMinimumIterations, salt, randomSetupPIN, setupPayload.setUpPINCode));
 
-        ReturnErrorOnFailure(
-            cluster.OpenCommissioningWindow(successCallback, failureCallback, timeout, ByteSpan(&verifier[0][0], sizeof(verifier)),
-                                            setupPayload.discriminator, kPBKDFMinimumIterations, salt, mPAKEVerifierID++));
+        ReturnErrorOnFailure(cluster.OpenCommissioningWindow(
+            successCallback, failureCallback, timeout, ByteSpan(reinterpret_cast<uint8_t *>(&verifier), sizeof(verifier)),
```

Comment:
Why a reinterpret cast here? It should be serialized as well, reverse of what was done elsewhere:

```
    memcpy(verifier.mW0, &verifierData[0], kSpake2p_WS_Length);
    memcpy(verifier.mL, &verifierData[kSpake2p_WS_Length], kSpake2p_WS_Length);
```

Reply:
The type of PASEVerifier changed to a structure, and passing the structure pointer as uint8_t * required the reinterpret cast. Alternatively, I can allocate a uint8_t array and memcpy the structure elements to it and pass that array as input to ByteSpan.

Reply:
That would be the long term safe way that is portable. The temp array can be on the stack as it is small enough